pkgs/tools/security/gnome-keysign/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchFromGitLab, python3, wrapGAppsHook
+, gobject-introspection, gpgme
+, gtk3, glib, gnome3, gst_all_1 }:
+
+let
+  # TODO: try to use multiple outputs
+  python-gpg = gpgme.override {
```

Comment:
Don't we have `python3Packages.gpgme` that does exactly this?

Reply:
Thanks.